src/groups/wrappers/z_bmq/z_bmqt/z_bmqt_messageguid.cpp

```diff
@@ -0,0 +1,98 @@
+#include <bmqt_messageguid.h>
+#include <bsl_sstream.h>
+#include <bsl_string.h>
+#include <z_bmqt_messageguid.h>
+
+int z_bmqt_MessageGUID__delete(z_bmqt_MessageGUID** messageGUID_obj)
+{
+    using namespace BloombergLP;
+
+    BSLS_ASSERT(messageGUID_obj != NULL);
+
+    bmqt::MessageGUID* messageGUID_p = reinterpret_cast<bmqt::MessageGUID*>(
+        *messageGUID_obj);
+    delete messageGUID_p;
+    *messageGUID_obj = NULL;
+
+    return 0;
+}
+
+int z_bmqt_MessageGUID__create(z_bmqt_MessageGUID** messageGUID_obj)
+{
+    using namespace BloombergLP;
+
+    bmqt::MessageGUID* messageGUID_p = new bmqt::MessageGUID();
+    *messageGUID_obj = reinterpret_cast<z_bmqt_MessageGUID*>(messageGUID_p);
+
+    return 0;
+}
+
+int z_bmqt_MessageGUID__fromBinary(z_bmqt_MessageGUID*  messageGUID_obj,
+                                   unsigned const char* buffer)
```

Comment:
This implementation is incorrect. It does not use `buffer` at all...

Reply:
changed but needs testing / review